src/parser/csql_grammar.y

```diff
@@ -150,6 +150,8 @@ static void pt_fill_conn_info_container(PARSER_CONTEXT *parser, int buffer_pos,
 #include "memory_alloc.h"
 #include "db_elo.h"
 #include "storage_common.h"
+#include "jsp_cl.h"
```

Comment:
What does it refer by the header file? The header file is changed?

Reply:
The build works fine without jsp_cl.h.

Reply:
Ok, thank you. I'll remove the header.